app/assets/javascripts/reportcard/reportcard.js

```diff
@@ -1,220 +1,228 @@
-                return Treehouse.__super__.getData.call(this, {
-                    dataType: 'json'
-                });
-            };
+                Treehouse.prototype.getData = function() {
+                    return Treehouse.__super__.getData.call(this, {
+                        dataType: 'json'
+                    });
+                };
 
-            Treehouse.prototype.transform = function(data) {
-                return {
-                    site: "Treehouse",
-                    username: data.profile_name,
-                    profile_url: data.profile_url,
-                    points: data.points,
-                    points_total: data.points.total,
-                    badge_count: data.badges.length,
-                    badges: data.badges.map(function(badge) {
-                        return {
-                            courses: badge.courses,
-                            course_count: badge.courses.length,
-                            earned_date: Date.parse(badge.earned_date),
-                            icon_url: badge.icon_url,
-                            label: badge.name,
-                            url: badge.url
-                        };
-                    })
+                Treehouse.prototype.transform = function(data) {
+                    return {
+                        site: "Treehouse",
+                        username: data.profile_name,
+                        profile_url: data.profile_url,
+                        points: data.points,
+                        points_total: data.points.total,
+                        badge_count: data.badges.length,
+                        badges: data.badges.map(function(badge) {
+                            return {
+                                courses: badge.courses,
+                                course_count: badge.courses.length,
+                                earned_date: Date.parse(badge.earned_date),
+                                icon_url: badge.icon_url,
+                                label: badge.name,
+                                url: badge.url
+                            };
+                        })
+                    };
                 };
-            };
 
-            return Treehouse;
+                return Treehouse;
 
-        })(ReportCard);
-        Codeschool = (function(_super) {
-            __extends(Codeschool, _super);
+            })(ReportCard);
+            Codeschool = (function(_super) {
+                __extends(Codeschool, _super);
 
-            function Codeschool(params) {
-                Codeschool.__super__.constructor.call(this, params);
-                this.url = "https://www.codeschool.com/users/" + this.userName + ".json";
-            }
+                function Codeschool(params) {
+                    Codeschool.__super__.constructor.call(this, params);
+                    this.url = "https://www.codeschool.com/users/" + this.userName + ".json";
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Too many errors. (75% scanned).